### DISCUSSION TITLE
Is it legal to use the Microsoft Edge/Windows service without an API key?

### DISCUSSION THREAD

**Discussion by matiashohn** [rany2/edge-tts#89] [General]
Is it legal to use the 'Microsoft Edge/Windows' service without an API key? I'm interested in using it to access Microsoft Edge's online text-to-speech service from Python, but I want to make sure I comply with the terms and conditions. Thank you

**Comment by rany2** [comment]
I don't know and besides, I'm not a lawyer so even if I read the TOS I won't be able to understand it. Though as far as I know and judging by the fact that there are similar libraries (like gTTS) I think it should be fine.
